examples/non_cartesian_reconstruction_stack_3d.py

```diff
@@ -1,122 +0,0 @@
-"""
```

Comment:
I think we should keep this example and only run it if certain conditions are met. For example it could be:
- sufficient number of cpu cores
- not being in CI

I guess all of these conditions could be checked via environment variables

Reply:
yes we can do that I think

Reply:
Back at this, I feel this will take longer, can we merge for now and get at this later? esp as this blocks #116 #114 ?

Reply:
Ok, let's then open an issue to reintegrate this, where we can copy-paste the content of this file.